plutus-playground-client/src/MainFrame.purs

```diff
@@ -359,14 +407,18 @@ handleAction CompileProgram = do
     Just contents -> do
       oldCompilationResult <- use _compilationResult
       assign _compilationResult Loading
+      assign (_editorState <<< _feedbackPaneMinimised) false
       newCompilationResult <- postContract contents
       assign _compilationResult newCompilationResult
-      -- If we got a successful result, update last compiled code and switch tab.
+      -- If we got a successful result, update lastCompiledCode and switch tab.
       case newCompilationResult of
         Success (Left _) -> pure unit
         _ -> do
-          updateCodeOnSuccess newCompilationResult mContents
-          replaceViewOnSuccess newCompilationResult Editor Simulations
+          -- next line commented out for now - I don't think we're doing this any more
+          -- replaceViewOnSuccess newCompilationResult Editor Simulations
```

Comment:
this is a FIXME I think, it should be resolved before PR merged

Reply:
Yep, I'm calling it. We don't do this any more.